### PR TITLE
[T5A1][W13-B4] Ang Wei Hao Desmond

### PR DESCRIPTION
**Pull request by KnewYouWereTrouble**

### CODE REVIEW

File: doc/DeveloperGuide.md [hunk 1]
```diff
@@ -74,12 +75,40 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Edit Tag
+
+**MSS**
+
+1. User requests to list persons
+2. AddressBook shows a list of persons
+3. User requests to edit a tag of a specific person in the list
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: there should be a confirmation interaction, address book asks for confirmation, then user confirms.

File: doc/DeveloperGuide.md [hunk 2]
```diff
@@ -74,12 +75,40 @@ Use case ends.
 > 3a1. AddressBook shows an error message <br>
   Use case resumes at step 2
 
+#### Use case: Edit Tag
+
+**MSS**
+
```
Comment (okkhoy) [inline doc/DeveloperGuide.md]: the activity was updated to rename a tag such that all people with the given tag will have the tag updated. please check the description again

**Comment by okkhoy** [comment]
@KnewYouWereTrouble 
Some comments added. Please ack & close the PR after reading comments.

**Comment by KnewYouWereTrouble** [comment]
Acknowledged

**Pull request closed without merging**